src/components/Navbar/LinkToClipboard.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
this shows as a new file, even though I previously made it, because I chaned it from roomView to navbar